lib/dev/ui/spinner.rb

```diff
@@ -31,7 +31,7 @@ class Task
           def initialize(title, &block)
             @title = title
             @thread = Thread.new do
-              cap = Dev::UI::StdoutRouter::Capture.new(with_frame_inset: false, &block)
+              cap = Dev::UI::StdoutRouter::Capture.new(self, with_frame_inset: false, &block)
```

Comment:
^ Note that I'm not totally convinced that's better but I like that it's more obvious what the intent is.

We should also make sure that the block arg isn't required in the task body. We can test it with `#arity?` if we need to but if you're splatting it, it might work already.